README.md

```diff
@@ -29,16 +47,20 @@ Your task is to create a RESTful API for a time clock system.  All required endp
 - Retrieve a list of employees
 - Clock in an employee
 - Clock out an employee
-- Retrieve a timesheet
 - Retrieve a list of timesheets
 
 ### Constraints
 
 - Employees must have a unique, valid SIN
+- An employee must be clocked in to be able to be clocked out
+
+### Search Criteria
+
+The list employees endpoint should support searching by first or last name. The timesheet list endpoint should support searching by employee ID and date range.
 
 ## Deliverables
 
-Provide a Github repository with your code, database schema and/or migrations, tests, and API documentation.  Include any setup steps necessary for your interviewer to be able to run your project.  Be prepared to review and explain any design decisions that you made.
+Provide a Github repository with your code, data schema, unit and feature tests, and API documentation.  Include a README with any setup steps necessary for your interviewer to be able to run your project.  Be prepared to screen share a working copy of your API with your interviewer and review and explain the design decisions that you made.  
```

Comment:
Remove that trailing whitespace.

I'd suggest enabling the option in your IDE to strip trailing whitespace automatically on save.